client/ionic/src/components/Carousel.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  IonContent,
+  IonPage,
+  IonSlides,
+  IonSlide,
+  IonCardSubtitle,
+  IonCardTitle,
+  IonImg,
+} from '@ionic/react';
+import 'tachyons';
+import useDynamicFlow from '../hooks/useDynamicFlow';
+import TopNav from '../components/TopNav';
+import { TextImageScreen, LoadedFlow } from '../content/flow';
+
+const TextImage = ({
+  screen,
+  flow,
+}: {
+  screen: TextImageScreen;
+  flow: LoadedFlow;
+}) => {
+  return (
+    <IonSlide>
+      <div className="flex flex-column justify-around h-100 pa4">
+        <div className="mt3 tl">
+          <div style={{ fontSize: '1.5em' }}>{screen.headingText}</div>
+          <div style={{ fontSize: screen.headingText ? '1em' : '1.5em' }}>
```

Comment:
clarifying - when there is a `headingText` in the screen, the font size of the body text is smaller?

Reply:
Yes. Initially, I was just going to make `headingText` equal to 1em as in the below image:

![image](https://user-images.githubusercontent.com/1689183/77375828-0a3a8600-6d45-11ea-855d-50396aa84543.png)

However, it appears that the splash carousel content also has a header and a long body (this wasn't in the mocks, but I decided to leave it as is) -- and changing the body text to be smaller made it look a lot better:

![image](https://user-images.githubusercontent.com/1689183/77375856-1c1c2900-6d45-11ea-9fd5-739dd7285b25.png)

Reply:
sounds good, thanks for clarifying! marking this as resolved